src/index.html

```diff
@@ -11,6 +11,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    
+    <div class="search-form-big">
```

Comment:
why do you need this wrapper you can just apply styles directly to the form element